sky/unit/test/gestures/scale_test.dart

```diff
@@ -36,7 +36,7 @@ void main() {
 
     TestPointer pointer1 = new TestPointer(1);
 
-    ui.PointerEvent down = pointer1.down(new Point(10.0, 10.0));
+    PointerInputEvent down = pointer1.down(new ui.Point(10.0, 10.0));
```

Comment:
Note, pre-merge, the file used a mix of ui.Point and Point. I'm not entirely sure why.